system-addon/content-src/components/TopSites/TopSite.jsx

```diff
@@ -21,25 +21,38 @@ export class TopSiteLink extends React.PureComponent {
    */
   _allowDrop(e, index) {
     let draggedIndex = parseInt(e.dataTransfer.getData("text/topsite-index"), 10);
-    if (!isNaN(draggedIndex) && draggedIndex !== index) {
+    if (!isNaN(draggedIndex)) {
```

Comment:
Looks like we don't use the `index` parameter anymore.. and we're basically just checking if `e.dataTransfer.types.includes("text/topsite-index")` now?

Reply:
yeah the only purpose of this now is to check that what's being dragged over is a top site and not something else. I guess it can just be `return !isNaN(parseInt(....));` or I could do some other property instead.